README.md

```diff
@@ -0,0 +1,27 @@
+This repo is home for the SPA (Single Page Application) project templates which ship as part of ASP.NET Core.
+At the moment, those are `Angular` and `React` templates.
+
+Besides these two templates, we also want this repo to serve as a knowledge-base for all community-owned SPA project templates out there.
+We expect to maintain that list in the `spaProjectTempltes.md` file. If you know of any such project templates, which are popular, send us a PR with added information in that file, and we'll include it, it it meets the popularity bar.
```

Comment:
nit: Capitalize S in `spaProjectTemplates.md`

Reply:
Actually, the file name is in lowercase, so I'd rather keep this as is.